app/js/states/inventory/inventory.jade

```diff
@@ -15,28 +11,11 @@ section.inventory
       .row
         .col-xs-6
           search-box(on-search='filter.setSearchTerm(model); filter.doFilter();', throttle='1000', placeholder='Find a system')
```

Comment:
`search-box` should not be placed within the elements that are hidden when the system list is reloading. Otherwise, when typing a longer expression into the search box the page starts to reload and the box disappears - making it impossible to finish typing